regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -80,3 +80,9 @@ <h3 class="toc-type">Table of Contents</h3>
 {% endblock %}
 
 {% block chrome-sidebar %}{% endblock %}
+
+{% block preload %}
+<script>
+  window.DOC_NUMBER = '{{ doc_number }}';
```

Comment:
I assume this is because it's easier to grab a global var than to traverse the dom for the attribute?